lib/testing/src/lib/process-services/pages/start-process.page.ts

```diff
@@ -34,6 +35,7 @@ export class StartProcessPage {
     processDefinitionOptionsPanel: ElementFinder = element(by.css('div[class*="mat-autocomplete-panel"]'));
 
     dropdownPage = new DropdownPage();
+    applicationDropdownPage = new DropdownPage(element(by.css('[data-automation-id*="start-process-app"] div[class="mat-select-arrow"]')));
```

Comment:
just a small improvement. Now that we have 2 dropdown we could rename to processDefinitionDropdown and applicationDropdownPage. There is no need to wrap the method like we do here https://github.com/Alfresco/alfresco-ng2-components/blob/74aadb162ecc2a83c364f83be4030db550b022a9/lib/testing/src/lib/process-services/pages/start-process.page.ts#L90 instead we could directly use `this.applicationDropdownPage.selectOption` and `this.processDefinitionDropdown.selectOption`

Reply:
I've renamed dropdownPage to processDefinitionDropdown and applicationDropdownPage to applicationDropdown. About the wrapping, it seems selectOption method is used in some tests too and there i wouldn't want to use the class member directly in the test.